app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala

```diff
@@ -33,8 +33,11 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   private val platformHost: Option[String] =
     config.getOptional[String]("platform.frontend.host")
 
+  private val localHelpFrontendBaseUrl =
+    config.getOptional[String]("help-frontend.base-url-local-testing")
+
   private val helpFrontendHost: String =
-    platformHost.map(_ => "").orElse(config.getOptional[String]("help-frontend.host")).getOrElse("")
+    platformHost.orElse(localHelpFrontendBaseUrl).getOrElse("")
```

Comment:
do you know why we only had a root relative path before rather than using platformHost? why we mapped it to "" - seems odd to do that and I'm wondering if there was some undocumented important reason for it or not

Reply:
I wonder if this would impact tracking consent frontend pages viewed on api platform - if after this - api platform pages that linked to pages on api platform, would start linking to www.tax.service.gov.uk urls

Reply:
Oh yes, good spot. That's probably why! I'll change it.